src/main/java/it/gov/pagopa/payhub/pdnd/model/PdndGenericConfig.java

```diff
@@ -0,0 +1,14 @@
+package it.gov.pagopa.payhub.pdnd.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class PdndGenericConfig {
```

Comment:
If it's not supposed to be created alone, let's make also this class abstract
```suggestion
public class PdndBaseServiceIntegratedConfig {
```